test/checkers.spec.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-console */
+const assume = require('assume');
+const checkers = require('../src/lib/checkers');
+const sinon = require('sinon');
+
+describe('Checkers', function () {
+
+  before(function () {
+    sinon.stub(console, 'log');
+  });
+
+  after(function () {
+    console.log.restore();
```

Comment:
```suggestion
    sinon.restore();
```

that way if you stub something else, it all gets unstubbed.

Reply:
Oh nice! I didn't know about that one. 👍🏻